### PR TITLE
Update src/util/awsAccountInfo.js to conditionally read hidden directory

### LINKED ISSUE #67: User still has to manually create config
After installation, the user still has to manually create the `~/.maestro/` directory and create the `aws_account_info.json` file to an empty object (`{}`). The `maestro config` or `maestro get-templates` command should handle this, or maybe there should be a separate `maestro init` or `maestro setup` command that does that. Regardless, errors should not be thrown by commands that expect the `aws_account_info.json` file to exist. Rather, a warning should be logged and the program exit early.

### PR DESCRIPTION
**Pull request by T-monius**
Fixes #67 .

Re-implement conditionally reading the hidden file of the home directory (as in PR #54).

### CODE REVIEW

File: src/util/awsAccountInfo.js [hunk 1]
```diff
@@ -1,8 +1,14 @@
 const fs = require("fs");
 const configDir = require("./configDir.js");
+const hiddenAccountFilePath = `${configDir}"aws_account_info.json"`;
```
Comment (johnisom) [inline src/util/awsAccountInfo.js:4]: `hiddenAccountFilePath` should be <code>\`${configDir}/aws_account_info.json\`</code>

Comment (johnisom) [reply src/util/awsAccountInfo.js:4]: On my machine it expands to `/home/john/.maestro"aws_account_info.json"`
when it should be `/home/john/.maestro/aws_account_info.json`.

Comment (zklamm) [reply src/util/awsAccountInfo.js:4]: Was just about to post the same thing.

Comment (johnisom) [reply src/util/awsAccountInfo.js:4]: Or it could be `configDir + "/aws_account_info.json"`; doesn't matter.

Comment (T-monius) [reply src/util/awsAccountInfo.js]: Fixed

**Comment by johnisom** [comment]
@T-monius The issue wasn't entirely solved. The double quotes are gone, but a slash (`/`) still needs to be between `configDir` and the filename.

**Pull request merged**